Tesserae/src/Components/ContextMenu.cs

```diff
@@ -126,30 +164,39 @@ public void ShowAt(int x, int y, int minWidth)
             }, 100);
         }
 
-        public void ShowFor(HTMLElement element, int distanceX = 1, int distanceY = 1)
+        public void ShowFor(HTMLElement element, int distanceX = 1, int distanceY = 1, bool asSubMenu = false)
```

Comment:
Can you make this one method internal, and only keep the public one without the bool asSubmenu flag?